doc/classes/InputEvent.xml

```diff
@@ -89,9 +89,9 @@
 			<param index="0" name="event" type="InputEvent" />
 			<param index="1" name="exact_match" type="bool" default="true" />
 			<description>
-				Returns [code]true[/code] if the specified [param event] matches this event. Only valid for action events i.e key ([InputEventKey]), button ([InputEventMouseButton] or [InputEventJoypadButton]), axis [InputEventJoypadMotion] or action ([InputEventAction]) events.
-				If [param exact_match] is [code]false[/code], it ignores additional input modifiers for [InputEventKey] and [InputEventMouseButton] events, and the direction for [InputEventJoypadMotion] events.
-				[b]Note:[/b] Only considers the event configuration (such as the keyboard key or joypad axis), not state information like [method is_pressed], [method is_released], [method is_echo], or [method is_canceled].
+				Returns [code]true[/code] if the specified [param event] matches this event. Only valid for action events, which include key ([InputEventKey]), button ([InputEventMouseButton] or [InputEventJoypadButton]), axis [InputEventJoypadMotion], and action ([InputEventAction]) events.
```

Comment:
I was genuinely just considering listing all of the valid classes, because summarising them as "_action events_" is a really bad way to put it.
Not only is "action event" not something consistently used in the docs to refer to this concept, but it can easily be misunderstood for **InputEventAction** (as it literally happens in this line)